src/mqueryfront/src/query/QueryMonaco.js

```diff
@@ -57,6 +57,12 @@ class QueryMonaco extends Component {
                         ]);
                     }
                 });
+                editor.addCommand(
+                    [monaco.KeyMod.CtrlCmd | monaco.KeyCode.Enter],
+                    () => {
+                        this.props.onSubmitQuery("medium");
```

Comment:
Why default to Medium instead of reading from the value of the combo box?

Reply:
AFAIR currently there's no combo box that allows you to choose priority before submit. Instead we have dropdown with buttons that are dispatching prioritized job.

Reply:
But question is still relevant: when we edit low-prioritized query, maybe the default priority should be also derived from the edited one? This change is still out of this PR's scope (we don't have a way to choose priority before submit), but maybe should be addressed by another issue.